### PR TITLE
add initial version of enhanced docs

### PR DESCRIPTION
**Pull request by alexsavio**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,36 +1,42 @@
...
-    on:
-      branch: master
+- provider: script
+  script: ci-scripts/semantic-release.sh
+  on:
+    branch: master
+
+notifications:
+  slack:
+    secure: u0VC5i9ja+4ib45MSyEsD0zcuWP/LOezTar1kvVQdTOIlfFFLF+CmXOY6Slxzk8SScBTv0+Lq7VgegLGnLWVWoBTZDhO8eM5H6e6StN7OpRVAMHuzbIOzMo0IyrNekSWxF5volDrnVH+3Ubi/S/v7kblmSKlwoWSb+H05Ft7XCWvKEuBYtomTpUfsAMQmORcIvm3cGQeBrcpanjYA4s3xE06RLVbbI2QStzobgH8awWb2fSx7zHXCVUb4wsQaIXCcrDGJ//5VqEYWB8gPIY5bD8QXewkRYPMJG6BILa4vBcc615SvUEje1UfaabJrXggGbV2FBKmQnEB9n0/9EaFWly1OR0yuOND2s7bsuFvAHBwaqZ/ZaW27WTtKQp5fmoSnOM8PoMI2BHqSijvqmkN0OKiFErmGaX1V9xppogiXCzQzurGdGm7TpbTwRHOHriSMF4n6B6CfblzMbaOwICnDq0QSJ/huMop8e36bzE3AzdpVaUHBjz8znfNnMEQUihO8SCm81PWEC235K1HsAS9YLnmRp66lJ0vpDgp4Bdd8gqxXeQl7AOgkYVXFzmWaDGcNsUldqK796SSf0Icoj4pzUn2cIvt7n19B2x8nIhWbc8+xt+JSvZcaSNzFOeeg2VI+h2KRt3cQE2BPv6x5BZFaMDQph5BPXqxXQI9tHAJdLc=
```
Comment (sleistner) [inline .travis.yml]: don't you want this in a travis env var?

File: README.md [hunk 2]
```diff
@@ -1,83 +1,165 @@
...
 ```
 
-You can also run the tasks separately with `make`:
 
-```bash
-make test
+Using jsonschema to validate a the input of a User model.
+
+```python
+from lambda_handler import validators, http_handler
```
Comment (sleistner) [inline README.md]: example would not work as `typing` import is missing

File: README.md [hunk 3]
```diff
@@ -1,83 +1,165 @@
 # lambda-handlers
 
-[![Version](https://img.shields.io/pypi/v/lambda-handlers.svg)](https://img.shields.io/pypi/v/lambda-handlers.svg)
-[![Build Status](https://travis-ci.org/enter-at/lambda-handlers.svg?branch=master)](https://travis-ci.org/enter-at/lambda-handlers)
-[![Maintainability](https://api.codeclimate.com/v1/badges/a39e55b85bfcc31204b9/maintainability)](https://codeclimate.com/github/enter-at/lambda-handlers/maintainability)
-[![Test Coverage](https://api.codeclimate.com/v1/badges/a39e55b85bfcc31204b9/test_coverage)](https://codeclimate.com/github/enter-at/lambda-handlers/test_coverage)
-[![Requirements Status](https://requires.io/github/enter-at/lambda-handlers/requirements.svg?branch=master)](https://requires.io/github/enter-at/lambda-handlers/requirements/?branch=master)
-[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
+[![](https://img.shields.io/pypi/v/lambda-handlers.svg)](https://pypi.org/project/lambda-handlers/)
```
Comment (sleistner) [inline README.md]: what is the reasoning in removing the titles?

Comment (alexsavio) [reply README.md]: when rendering to HTML the titles appeared next to the icons, this was the only way I found to not have them there. If you want, please check that with:
```
tox -e docs
open .tox/docs/tmp/html/index.html
```

Comment (sleistner) [reply README.md]: this s***s but ok...

File: .travis.yml (hunk 1 above)
Comment (alexsavio) [reply .travis.yml]: done. thx

File: README.md (hunk 2 above)
Comment (alexsavio) [reply README.md]: you're right! done

**Pull request merged**